leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -442,6 +442,16 @@
       excluded.instanceField("android.view.Choreographer$FrameDisplayEventReceiver",
           "mMessageQueue").alwaysExclude();
     }
+  },
+
+  SYSTEM_SENSOR_MANAGER_LENOVO {
```

Comment:
Please move next to the manufacturer specific ignores, and add a manufacturer boolean as well as a version boolean (see other entries in this class)